.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  docker-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}} 
+            type=semver,pattern={{major}}
+            type=ref,event=branch
+            type=sha
+            latest
+      - name: Publish Docker Image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          build-args: |
+            TARBALL_PATH=${{ needs.prepare.outputs.tarball-path }}
```

Comment:
nit: I actually don't think this is needed.

Reply:
i removed it, tested and it worked.